test/e2e/accounts/common.ts

```diff
@@ -78,6 +78,16 @@ export async function installSnapSimpleKeyring(
     tag: 'button',
   });
 
+  // wait for permissions popover, click checkboxes and confirm
```

Comment:
Why are there so many E2E changes in this PR?

Reply:
Because we changed permission weights, so now warning modal is displayed for some of them, that's an extra confirmation step which I needed to add.

Reply:
Reverted within the last changes made to the permission weights.